demo/actions.py

```diff
@@ -792,3 +797,60 @@ def run(self, dispatcher, tracker, domain):
             )
 
         return []
+
+
+class ActionTagConvo(Action):
+    """Returns the chitchat utterance dependent on the intent"""
+
+    def name(self):
+        return NotImplementedError
+
+    def tag_convo(self, tracker, label):
+        endpoint = f"http://{RASA_X_HOST}/api/conversations/{tracker.sender_id}/tags"
+        requests.post(url=endpoint, data=label)
+
+    def run(self, dispatcher, tracker, domain) -> List[EventType]:
+        return []
+
```

Comment:
I think we can just make this a `tag_convo` helper method outside of all of the action classes, can't we? instead of making a new action class for it

Reply:
sure, that works too